security/botan/Portfile

```diff
@@ -84,6 +106,13 @@ configure.optflags  -O3
 
 platform darwin { configure.args-append --os=darwin }
 
+# Variant to re-enable use of intrinsics
+variant native description {Enable all intrinsics} {
+    foreach elem $intrinsics {
+        configure.args-delete --disable-$elem
+    }
+}
+
```

Comment:
Love the approach you ultimately went with, eliminating potential copy-pasta. Great work!